bootstrap/kubeadm/controllers/kubeadmconfig_controller.go

```diff
@@ -97,7 +99,7 @@ func (r *KubeadmConfigReconciler) SetupWithManager(mgr ctrl.Manager, option cont
 			},
 		).
 		Watches(
-			&source.Kind{Type: &clusterv1.MachinePool{}},
+			&source.Kind{Type: &expv1.MachinePool{}},
```

Comment:
Should this be conditionalized on the feature flag being set?

Reply:
Done, thanks!